src/service/models.py

```diff
@@ -240,6 +240,10 @@ class Collection(BaseModel):
             }
         )
     ] = None
+    metadata: Annotated[dict[str, Any] | None, Field(
```

Comment:
Actually now I think about it, Any could include container types which I don't think we want
```suggestion
    metadata: Annotated[dict[str, str | int | float] | None, Field(
```